config/database.yml

```diff
@@ -1,15 +1,24 @@
 default: &default
   adapter: postgresql
   encoding: utf8
-  host: localhost
   pool: 10
   timeout: 5000
+  # username: <%= ENV['DB_USERNAME'] %>
+  # password: <%= ENV['DB_PASSWORD'] %>
+  # database: <%= ENV['DB_NAME'] %>
+  # host: <%= ENV['DB_HOST'] %>
+
```

Comment:
These added lines are commented out. Also the file isn't run through ERB as it is in Rails to the <%= %> substitutions won't work. Do we need to use these env vars in preference to static values? If so it's probably best to tweak another part of the system so this file is run through ERB before being evaluated by YAML.